eth/downloader/queue.go

```diff
@@ -837,6 +841,9 @@ func (q *queue) DeliverBodies(id string, txLists [][]*types.Transaction, txListH
 		}
 
 		// do some sanity check for sidecar
+		if blobTxs != len(sidecars[index]) {
```

Comment:
when do full sync,
if distance between the target block and the latest block is larger than MinBlocksForBlobRequests, then the blob data is not need
this check will reject the target block in the wrong way

Reply:
Same answer as below, I think it's a good thing to not mark a peer as invalid if some sidecars are missing. I could add the check for `MinBlocksForBlobRequests` here though, WDYT?

Reply:
> Same answer as below, I think it's a good thing to not mark a peer as invalid if some sidecars are missing. I could add the check for `MinBlocksForBlobRequests` here though, WDYT?

👌